CMSgov/mint-app
# Easi 1961/model discussion center

# EASI-1961

## Changes and Description

- Create ReactModal component for Discussion center
- Created task list banner for Discussions statuses
- Mobile responsive
- Unit and Cypress testing

Can be accessed from the task list or any form page - minus collaborator and documents
Unanswered questions are sorted from oldest to newest
Answered questions are sorted from newest reply to oldest

![Screen Shot 2022-05-24 at 1 08 24 PM](https://user-images.githubusercontent.com/95709965/170093002-4b342292-c8d5-46b1-96ec-e06cce05afde.png)

## How to test this change

- Create a model plan
- From task list, ask a question
- Click to answer the question and answer it
- See the question is marked as answered and moved to the answered accordion
- Repeat for `Ask a question` links on form pages
- Test for mobile responsive

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
